### PR TITLE
12 tariff

### PR DESCRIPTION
**Pull request by romigos**

### CODE REVIEW

File: app/views/tariff_groups/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<br>
+<br>
+<h1>Всі категорії тарифів</h1>
+<table style="width:100%;" border="1">
+  <tr>
+    <td width="50%">Назва категорії тарифу</td>
+
```
Comment (saviokmua) [inline app/views/tariff_groups/index.html.erb]: @romigos why here this empty line?

Comment (romigos) [reply app/views/tariff_groups/index.html.erb]: sorry fixed

File: app/controllers/format_fotos_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class FormatFotosController < ApplicationController
```
Comment (saviokmua) [inline app/controllers/format_fotos_controller.rb]: @romigos FormatPhotosController

File: app/controllers/static_pages_controller.rb [hunk 3]
```diff
@@ -14,4 +14,4 @@ def conditions; end
   def faq; end
 
   def contacts; end
-end
+end
```
Comment (saviokmua) [inline app/controllers/static_pages_controller.rb]: @romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof

File: app/controllers/tariff_groups_controller.rb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  private
+
+  def bundle_tariff_group
+    @tariff_group = TariffGroup.find(params[:id])
+  end
+
+  def tariff_group_params
+    params.require(:tariff_group).permit(:name)
+  end
+end
```
Comment (saviokmua) [inline app/controllers/tariff_groups_controller.rb]: @romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof

File: app/controllers/tariffs_controller.rb [hunk 5]
```diff
@@ -42,4 +42,4 @@ def bundle_tariff
   def tariff_params
     params.require(:tariff).permit(:name, :price)
   end
-end
+end
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb]: @romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof

File: app/helpers/format_fotos_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module FormatFotosHelper
```
Comment (saviokmua) [inline app/helpers/format_fotos_helper.rb]: @romigos please remove empty modules

File: app/views/format_fotos/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<%= form_for :format_foto, url: format_fotos_path do |f| %>
+  <p>
+    <%= f.label :name, 'Назва формату фото' %>
+    <%= f.text_field(:name, { class: 'form-control' }) %>
+  </p>
+  <p>
+    <br>
+    <%= f.submit('Зберегти', { class: 'btn btn-success' }) %>
+  </p>
+<% end %>
```
Comment (saviokmua) [inline app/views/format_fotos/_form.html.erb]: @romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof 
and next places

File: app/views/format_fotos/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<div id="tariff">
+<h1>Всі формати фото</h1>
+<table style="width:100%;" border="1">
```
Comment (saviokmua) [inline app/views/format_fotos/index.html.erb]: @romigos all styles please move to css classes

**Review by saviokmua — CHANGES_REQUESTED** [review]

File: app/controllers/static_pages_controller.rb (hunk 3 above)
Comment (romigos) [reply app/controllers/static_pages_controller.rb]: fixed

File: app/controllers/tariffs_controller.rb (hunk 5 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb]: fixed

File: app/controllers/tariff_groups_controller.rb (hunk 4 above)
Comment (romigos) [reply app/controllers/tariff_groups_controller.rb]: fixed

File: app/views/format_fotos/_form.html.erb (hunk 7 above)
Comment (romigos) [reply app/views/format_fotos/_form.html.erb]: fixed

File: app/views/format_fotos/index.html.erb (hunk 8 above)
Comment (romigos) [reply app/views/format_fotos/index.html.erb]: fixed

File: app/controllers/format_fotos_controller.rb (hunk 2 above)
Comment (saviokmua) [reply app/controllers/format_fotos_controller.rb]: @romigos why you ignored this?

Comment (romigos) [reply app/controllers/format_fotos_controller.rb]: fixed

File: app/helpers/format_fotos_helper.rb (hunk 6 above)
Comment (romigos) [reply app/helpers/format_fotos_helper.rb]: fixed

File: app/controllers/format_photos_controller.rb [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+class FormatPhotosController < ApplicationController
+  before_action :bundle_format_photo, only: %i[show edit update destroy]
+
+  def index
+    @format_photo = FormatPhoto.all
+  end
+
+  def new
+    @tformat_photo = FormatPhoto.new
```
Comment (saviokmua) [inline app/controllers/format_photos_controller.rb]: what is mean tformat_photo?

please listen https://www.buzzsprout.com/238868/899519- about this problem

File: app/controllers/tariffs_controller.rb (hunk 5 above)
Comment (saviokmua) [reply app/controllers/tariffs_controller.rb]: image

Comment (saviokmua) [reply app/controllers/tariffs_controller.rb]: @romigos it is not true

**Review by saviokmua — CHANGES_REQUESTED** [review]

File: app/controllers/format_photos_controller.rb (hunk 9 above)
Comment (romigos) [reply app/controllers/format_photos_controller.rb]: fixed

File: app/controllers/tariffs_controller.rb (hunk 5 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb]: fixed

File: app/controllers/format_fotos_controller.rb (hunk 2 above)
Comment (romigos) [reply app/controllers/format_fotos_controller.rb]: fixed

File: app/views/format_fotos/index.html.erb (hunk 8 above)
Comment (saviokmua) [reply app/views/format_fotos/index.html.erb]: @romigos 
where???
and app/views/format_fotos/index.html.erb => app/views/format_**photos**/index.html.erb

**Review by saviokmua — CHANGES_REQUESTED** [review]

**Comment by romigos** [comment]
sorry i do not delete some old files

File: app/views/format_fotos/index.html.erb (hunk 8 above)
Comment (romigos) [reply app/views/format_fotos/index.html.erb]: fixed file deleted

File: app/helpers/format_fotos_helper.rb (hunk 6 above)
Comment (romigos) [reply app/helpers/format_fotos_helper.rb]: fixed

**Pull request merged**